cclib/io/moldenwriter.py

```diff
@@ -43,6 +43,10 @@ def _title(self, path):
     def _coords_from_ccdata(self, index):
         """Create [Atoms] section using geometry at the given index."""
         elements = [self.pt.element[Z] for Z in self.ccdata.atomnos]
+        if hasattr(self,'ghost'):
```

Comment:
```suggestion
        if hasattr(self, 'ghost'):
```
or, if you do the one-liner, `if self.ghost is not None`.